node/nodeTransactions_test.go

```diff
@@ -376,14 +379,24 @@ func TestNode_ComputeTransactionStatusAllBranches(t *testing.T) {
 	txStatus = n.ComputeTransactionStatus(normalTxCrossShard, true)
 	assert.Equal(t, core.TxStatusPartiallyExecuted, txStatus)
 
-	// cross shard scr in storage source shard
+	// intra shard scr in storage source shard
 	shardCoordinator.SelfShardId = 0
 	txStatus = n.ComputeTransactionStatus(unsignedTxIntraShard, false)
+	assert.Equal(t, core.TxStatusExecuted, txStatus)
+
+	// intra shard scr in pool source shard
+	shardCoordinator.SelfShardId = 0
+	txStatus = n.ComputeTransactionStatus(unsignedTxIntraShard, true)
+	assert.Equal(t, core.TxStatusReceived, txStatus)
+
+	// cross shard scr in storage source shard
```

Comment:
Cross-shard, on destination, for rewards - not covered. But the test is sufficient though.

Reply:
yeah i think is sufficient because all branches are covered.